packages/plugins/src/plugins/eigenlayer/constants.ts

```diff
@@ -0,0 +1,19 @@
+export const platformId = 'eigenlayer';
+export const platform = {
+  id: platformId,
+  name: 'Eigenlayer',
+};
+
+export const chain = 'ethereum';
+
+const eigenTokenAddress = '0xec53bF9167f50cDEB3Ae105f56099aaaB9061F83';
```

Comment:
```suggestion
const eigenTokenAddress: Address = '0xec53bF9167f50cDEB3Ae105f56099aaaB9061F83';
```

We should explicitly _up-scale_ the type here; see https://github.com/Octav-Labs/sonarwatch-portfolio/blob/74f39eb4a04d1b5e9a89e4c4b3abc5e3a2c3165e/packages/plugins/src/plugins/stader/constants.ts#L12-L13 as an example.